include/internal/libfreenect2/rgb_packet_processor.h

```diff
@@ -108,5 +109,22 @@ class VTRgbPacketProcessor : public RgbPacketProcessor
 };
 #endif
 
+#ifdef LIBFREENECT2_WITH_VAAPI_SUPPORT
+class VaapiRgbPacketProcessorImpl;
+
+class VaapiRgbPacketProcessor : public RgbPacketProcessor
+{
+public:
+  VaapiRgbPacketProcessor();
+  virtual ~VaapiRgbPacketProcessor();
+  virtual bool good();
+  virtual Allocator *getAllocator();
+protected:
+  virtual void process(const libfreenect2::RgbPacket &packet);
```

Comment:
btw. why are all process methods of all rgb packet processor implementations protected, the process method is public in the PacketProcessor class